### DISCUSSION TITLE
Faild build with ant

### DISCUSSION THREAD

**Discussion by mutalibcs** [community/community#23540] [New to GitHub]
<p>In my repo I have tried to build my project Java with Ant. But, it shows an error. Here, I attached the photo.<br>
Here, my repository: <a href="https://github.com/mamutalib/Scientific_Calculator" rel="noopener nofollow ugc">My Repository: Scientific Calculator</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/1/819485244e740543a4a1ff6d76f9ebad29b78b4d.png" data-download-href="/uploads/short-url/iujNnKYjsd63nUgcKhdDk3lktcx.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/61675569/181077807-c487637c-9ab4-4a34-84d9-cfe567fabc29.png" alt="image" data-base62-sha1="iujNnKYjsd63nUgcKhdDk3lktcx" width="690" height="431" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/1/819485244e740543a4a1ff6d76f9ebad29b78b4d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">761×476 20.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by mutalibcs** [comment]
<p>Will I get any solution from here!?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>There’s only a tiny part of the error visible in your screenshot, but javac saying “package […] does not exist” sounds like there are some missing dependencies. Make sure to install whatever your project needs.</p>
